sova/roles/triage.py

```
@@ -361,9 +364,50 @@ async def assess_task_with_llm(self, task: Task, ctx: ExecutionContext) -> TaskA
             )
-            assessment = self._parse_llm_assessment(result.text)
-            if assessment:
-                return assessment
+
+            # Validate output with retry on failure
+            try:
+                triage_budget = ctx.config.agent.max_budget / 10
+                triage_spent = result.cost_usd
+                last_retry_text = result.text
+
+                async def retry_triage(retry_prompt: str):
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `retry_triage`

(ANN202)